platform_versioned_docs/version-24.1.0/credentials/managed_identities.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: "Managed identities"
+description: "Configure managed identities in Seqera Platform."
+date: "12 Jun 2024"
+tags: [organizations, administration, managed identities, clusters, ssh, credentials]
+---
+
+With traditional SSH credentials in Seqera Platform, all workspace users access HPC clusters with the same set of credentials configured in the workspace. While this simplifies access control to computing resources, all user activity on your HPC cluster is logged under the same user credentials. This also means that all Seqera workspace users have the same access permissions on your HPC cluster. When multiple workspace users run pipelines and consume compute resources, all activity is reflected under the same user on your HPC cluster. Seqera managed identities resolve this limitation by enabling granular access control to HPC clusters for individual workspace users. 
+
+Configure a managed identity at the organization level to be used to access the supported HPC compute environments in your organization workspaces. A managed identity is selected for authentication to your compute environment in the same way that a traditional credential would be selected, but it contains multiple user credentials each tied to a unique Seqera user. This preserves the identity of the user launching workflows on your HPC cluster or compute environment, thereby improving traceability and adherence to data access policies. 
```

Comment:
```suggestion
You can configure a managed identity at the organization level for it to be used to access the supported HPC compute environments in your organization workspaces. 

A managed identity is selected for authentication to your compute environment in the same way that a traditional credential is be selected, but it contains multiple user credentials each tied to a unique user. This preserves the identity of the user launching workflows on your HPC cluster or compute environment, improving traceability and adherence to data access policies. 
```

Reply:
We avoid "you can" and use simpler, more direct language.